helmclient.go

```diff
@@ -394,6 +395,47 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 	return nil
 }
 
+// RunReleaseTest runs the tests for a Helm Release. The releaseName is the
+// name of the Helm Release that is set when the Helm Chart is installed. This
+// is the same action as running the helm test command.
+func (c *Client) RunReleaseTest(releaseName string, options ...helmclient.ReleaseTestOption) error {
+	c.logger.Log("level", "debug", "message", fmt.Sprintf("running tests for release '%s'", releaseName))
+
+	t, err := c.newTunnel()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	defer c.closeTunnel(t)
+
+	resChan, errChan := c.newHelmClientFromTunnel(t).RunReleaseTest(releaseName, helmclient.ReleaseTestTimeout(300))
+	if IsReleaseNotFound(err) {
+		return backoff.Permanent(microerror.Maskf(releaseNotFoundError, releaseName))
```

Comment:
What does `backoff.Permanent` do here?

Reply:
Godoc says it wraps the error in a permanent error. It's used in all the other methods so I used it for consistency.